saidump/saidump.cpp

```diff
@@ -56,15 +18,14 @@ void printUsage()
     std::cout << "        Print out this message" << std::endl;
 }
 
-CmdOptions handleCmdLine(int argc, char **argv)
+sai_status_t SaiDump::handleCmdLine(int argc, char **argv)
```

Comment:
since this is changed to class now, please also rename file from saidump.cpp to SaiDump.cpp, all files should follow the same naming convention

Reply:
Fixed.